ScoreSaber/Patches/LeaderboardPatches.cs

```diff
@@ -92,7 +93,7 @@ internal class LeaderboardPatches : IInitializable, IAffinity {
                     new DataItem(____globalLeaderboardIcon, "Global"),
                     new DataItem(____aroundPlayerLeaderboardIcon, "Around You"),
                     new DataItem(____friendsLeaderboardIcon, "Friends"),
-                    new DataItem(_countryIcon, "Country"),
+                    new DataItem(_countryIcon, "Location"),
```

Comment:
I think this would be nicer if it switched between "Country" and "Region", depending on your settings

Reply:
Should the icon change too? It would be fun to see if these can be changed as well

Reply:
Also, what should these icons be 🤔